components/link/index.js

```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+
+const StyledLink = styled.a`
+  color: ${(props) => props.theme.color.copy.light};
+`
+
+const Link = ({ children, href }) => (
```

Comment:
I'm not sure having a custom <Link> component is necessary. I'd use the Next.js `Link` component directly and I'd set the color globally, or on the parent component. Thoughts?

Reply:
Makes sense, I'll refactor it